include/crm/msg_xml_compat.h

```diff
@@ -46,6 +46,9 @@ extern "C" {
 //! \deprecated Do not use (will be removed in a future release)
 #define XML_ATTR_RA_VERSION "ra-version"
 
+//! \deprecated Do not use (will be removed in a future release)
+#define XML_CIB_ATTR_REPLACE "replace"
```

Comment:
Moving this back to `msg_xml.h` for use in a piece of compatibility code, keeping it deprecated though